### ISSUE TITLE
Remove limit of 20 results in Nearby Venue Search

### ISSUE THREAD

**Issue by Pohli** [rinigus/pure-maps#293]
Example: If I search for Gas Stations nearby Dortmund I get always 20 results whatever distance I've selected (I tried 5, 10 and 20 km).

**Comment by rinigus** [comment]
Yes, but these stations should be sorted according to the distance from point that you search from. So, what would be a reasonable limit in your opinion?

**Comment by Pohli** [comment]
I suggest to keep the limit of 20 results for the initial search but add an Show/Load More option in a bottom pulley menu on the result page. Btw.: Maybe 20 results would be enough if I could search for a specific gas station brand but that's another topic.

**Comment by rinigus** [comment]
Nearby search also has "Name" option. Maybe use that. For providers that don't have separate Name, you could just use Type as a Name

**Comment by Pohli** [comment]
I checked it:
1. Default search provider Foursqare (I use this usually): no Name field, typed brand name "Esso" into Type field: no results.
2. Search provider MapQuest Nominatim: no Name field, typed brand name into Type field: 6 valid results.
3. Search provider OSM Scout (has Name field): selected "Gas Station", "Gas Stations" and "Any" for Type field, typed brand name into Name field: always the same 4 valid results.
Selected 10 km Radius for all cases.
I don't like to use Search provider OSM Scout so much cause the search takes much longer than the others on my Jolla Phone.

Anyway if one wants to search for any gas stations the 20 results limit would still occur. Would do you think about the above mentioned Load More option?

**Comment by rinigus** [comment]
"Load more" is somewhat harder to implement - its a new call. Not impossible though.

I may look into how to specify name for Foursquare - its probably  a better solution in longer term
